pom.xml

```diff
@@ -142,8 +148,9 @@
                 <artifactId>maven-hpi-plugin</artifactId>
 				<!--  <version>1.67</version> -->
                 <extensions>true</extensions>
-                <configuration>
-                    <compatibleSinceVersion>0.2</compatibleSinceVersion>
+		<configuration>
+			<pluginFirstClassLoader>true</pluginFirstClassLoader>
+			<!-- <compatibleSinceVersion>0.2</compatibleSinceVersion> -->
```

Comment:
Why was this commented out?

Reply:
Sorry, I forgot to uncoment it. I'll fix this.